loader_hub/library.json

```diff
@@ -121,6 +121,15 @@
       "apollo"
     ]
   },
+  "GraphReader": {
+    "id": "graph",
```

Comment:
ah you have to name this the same as the directory name "graphdb_cypher"

Reply:
Done!